ompi/group/group.h

```diff
@@ -337,15 +340,14 @@ int ompi_group_minloc (int list[], int length);
  * @brief Helper function for retreiving the proc of a group member in a dense group
  *
  * This function exists to handle the translation of sentinel group members to real
- * ompi_proc_t's. If a sentinel value is found and allocate is true then this function
+ * ompi_proc_t's. If a sentinel value is found and allocate bit is true then this function
  * looks for an existing ompi_proc_t using ompi_proc_for_name which will allocate a
- * ompi_proc_t if one does not exist. If allocate is false then sentinel values translate
+ * ompi_proc_t if one does not exist. If allocate bit is false then sentinel values translate
  * to NULL.
  */
-static inline struct ompi_proc_t *ompi_group_dense_lookup (ompi_group_t *group, const int peer_id, const bool allocate)
+static inline struct ompi_proc_t *ompi_group_dense_lookup (ompi_group_t *group, const int peer_id, unsigned char flags)
```

Comment:
I think this can still be a const?

Reply:
since the passed value is not a pointer, I guess `const` can safely be ignored,